### ISSUE TITLE
Accessibility problems with screen readers

### ISSUE THREAD

**Issue by ethindp** [turt2live/matrix-dimension#465]
The entire dimension interface (under Room Options -> Widgets) has accessibility problems with screen readers. As an example, I have a matrix room with two Jitsi widgets, one of which is unpinned. I cannot see the unpinned widget in the integrations manager, nor can I seem to delete it. Additionally, when I open the widget admin page to add new widgets, I can't close it with my screen reader, and my only recourse is to restart element entirely. It is a bit better on Android, but the inability to remove unpinned widgets is still an issue. Is there a way I can get around these problems?
